### PR TITLE
Follow not published daemonset images

### PR DESCRIPTION
**Pull request by kenhys**
Dockerfile itself is still maintained, but not published state
is troublesome, we need to explain it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,13 @@ See also dockerhub tags page: https://hub.docker.com/r/fluent/fluentd-kubernetes
...
+As the current limitation about the number of automated builds on hub.docker.com, there are some restrictions to ship daemonset images:
+
+* `stackdriver`, `papertrail`, `syslog` images (x86_64/arm64) won't be published anymore
+* `logentries`, `loggy`, `logzio`, `s3` arm64 images won't be published anymore (x86_64 only supported)
+
+If you want to use above non published images, build it by yourself. Dockefile itself is still maintained in this repository.
+
 ##### Multi-Arch images
 - `Azureblob`
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13.1-debian-azureblob-1.0`
```
Comment (cosmo0920) [inline README.md:25]: loggly and logentries are also not multiarch-ed images.

File: README.md [hunk 2]
```diff
@@ -33,15 +40,9 @@ See also dockerhub tags page: https://hub.docker.com/r/fluent/fluentd-kubernetes
 - `Cloudwatch`
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13.1-debian-cloudwatch-1.0`
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13-debian-cloudwatch-1`
-- `Stackdriver`
-  - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13.1-debian-stackdriver-1.0`
-  - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13-debian-stackdriver-1`
 - `S3`
```
Comment (cosmo0920) [inline README.md:43]: s3 is not multiarched. Only for amd64:
https://hub.docker.com/layers/fluent/fluentd-kubernetes-daemonset/v1.13.1-debian-s3-amd64-1.0/images/sha256-b5a8dd30151795a25e477a34152e1d177ba8492098f8783c0f4ab12e18dd56da?context=repo

File: README.md [hunk 3]
```diff
@@ -51,9 +52,6 @@ See also dockerhub tags page: https://hub.docker.com/r/fluent/fluentd-kubernetes
 - `Graylog`
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13.1-debian-graylog-1.0`
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13-debian-graylog-1`
-- `Papertrail`
-  - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13.1-debian-papertrail-1.0`
-  - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13-debian-papertrail-1`
 - `Logzio`
```
Comment (cosmo0920) [inline README.md:55]: s3 is not multiarched. Only for amd64:
https://hub.docker.com/layers/fluent/fluentd-kubernetes-daemonset/v1.13.1-debian-logzio-amd64-1.0/images/sha256-49e70c93e717d18b83808c5a3cd20cd11871514a9930cf66f76aa87cdbfb3b6f?context=repo

**Review by cosmo0920 — CHANGES_REQUESTED** [review]
I found that loggly, logentries, S3, and logzio are not multiarched images.
We should drop these entries from Multi-Arch images section.

File: README.md (hunk 1 above)
Comment (kenhys) [reply README.md:25]: Thanks, fixed.

File: README.md (hunk 2 above)
Comment (kenhys) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (kenhys) [reply README.md]: fixed

File: templates/README.md.erb [hunk 4]
```diff
@@ -17,9 +17,17 @@ See also dockerhub tags page: https://hub.docker.com/r/fluent/fluentd-kubernetes
...
+
+* `stackdriver`, `papertrail`, `syslog` images (x86_64/arm64) won't be published anymore
+* `logentries`, `loggly`, `logzio`, `s3` arm64 images won't be published anymore (x86_64 only supported)
+
+If you want to use above non published images, build it by yourself. Dockefile itself is still maintained in this repository.
+
 ##### Multi-Arch images
 <% debian.each do |image| %>
 <% path = image.split(":").first.sub("-amd64",""); tags = image.split(":").last %>
+<% next if %w(stackdriver papertrail syslog logentries loggly logzio s3).any? do |component| tags.include?(component) end %>
```
Comment (cosmo0920) [inline templates/README.md.erb:30]: Can we generate this definition from x86_64 and arm64 skip image name array?
I think that it is redundant to write here. Instead we should generate from skipable image name definitions.

Comment (cosmo0920) [reply templates/README.md.erb]: Fixed in https://github.com/fluent/fluentd-kubernetes-daemonset/pull/581/commits/262f82924833547bf59b33c1ebe9eebc63ebb14d.

**Review by cosmo0920 — APPROVED** [review]
Let's :shipit:!

**Comment by cosmo0920** [comment]
And I'm OK to ship as-is.
But I think that x86_64 and arm64 section still needs to display Dockerfile paths for non built images.
What do you think?

**Comment by cosmo0920** [comment]
Like following changes:

```diff
diff --git a/README.md b/README.md
index f9c4f00..ba71185 100644
--- a/README.md
+++ b/README.md
@@ -73,9 +73,11 @@ If you want to use above non published images, build it by yourself. Dockefile i
 - `Cloudwatch` [Dockerfile](docker-image/v1.13/debian-cloudwatch/Dockerfile)
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13.1-debian-cloudwatch-amd64-1.0`
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13-debian-cloudwatch-amd64-1`
+- `Stackdriver` [Dockerfile](docker-image/v1.13/debian-stackdriver/Dockerfile)
 - `S3` [Dockerfile](docker-image/v1.13/debian-s3/Dockerfile)
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13.1-debian-s3-amd64-1.0`
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13-debian-s3-amd64-1`
+- `Syslog` [Dockerfile](docker-image/v1.13/debian-syslog/Dockerfile)
 - `Forward` [Dockerfile](docker-image/v1.13/debian-forward/Dockerfile)
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13.1-debian-forward-amd64-1.0`
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13-debian-forward-amd64-1`
@@ -85,6 +87,7 @@ If you want to use above non published images, build it by yourself. Dockefile i
 - `Graylog` [Dockerfile](docker-image/v1.13/debian-graylog/Dockerfile)
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13.1-debian-graylog-amd64-1.0`
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13-debian-graylog-amd64-1`
+- `Papertrail` [Dockerfile](docker-image/v1.13/debian-papertrail/Dockerfile)
 - `Logzio` [Dockerfile](docker-image/v1.13/debian-logzio/Dockerfile)
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13.1-debian-logzio-amd64-1.0`
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13-debian-logzio-amd64-1`
@@ -109,9 +112,14 @@ If you want to use above non published images, build it by yourself. Dockefile i
 - `Elasticsearch6` [Dockerfile](docker-image/v1.13/arm64/debian-elasticsearch6/Dockerfile)
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13.1-debian-elasticsearch6-arm64-1.0`
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13-debian-elasticsearch6-arm64-1`
+- `Loggly` [Dockerfile](docker-image/v1.13/arm64/debian-loggly/Dockerfile)
+- `Logentries` [Dockerfile](docker-image/v1.13/arm64/debian-logentries/Dockerfile)
 - `Cloudwatch` [Dockerfile](docker-image/v1.13/arm64/debian-cloudwatch/Dockerfile)
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13.1-debian-cloudwatch-arm64-1.0`
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13-debian-cloudwatch-arm64-1`
+- `Stackdriver` [Dockerfile](docker-image/v1.13/arm64/debian-stackdriver/Dockerfile)
+- `S3` [Dockerfile](docker-image/v1.13/arm64/debian-s3/Dockerfile)
+- `Syslog` [Dockerfile](docker-image/v1.13/arm64/debian-syslog/Dockerfile)
 - `Forward` [Dockerfile](docker-image/v1.13/arm64/debian-forward/Dockerfile)
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13.1-debian-forward-arm64-1.0`
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13-debian-forward-arm64-1`
@@ -121,6 +129,8 @@ If you want to use above non published images, build it by yourself. Dockefile i
 - `Graylog` [Dockerfile](docker-image/v1.13/arm64/debian-graylog/Dockerfile)
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13.1-debian-graylog-arm64-1.0`
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13-debian-graylog-arm64-1`
+- `Papertrail` [Dockerfile](docker-image/v1.13/arm64/debian-papertrail/Dockerfile)
+- `Logzio` [Dockerfile](docker-image/v1.13/arm64/debian-logzio/Dockerfile)
 - `Kafka` [Dockerfile](docker-image/v1.13/arm64/debian-kafka/Dockerfile)
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13.1-debian-kafka-arm64-1.0`
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13-debian-kafka-arm64-1`
diff --git a/templates/README.md.erb b/templates/README.md.erb
index 2594e01..a7adf7d 100644
--- a/templates/README.md.erb
+++ b/templates/README.md.erb
@@ -40,8 +40,8 @@ If you want to use above non published images, build it by yourself. Dockefile i
 ##### x86_64 images
 <% debian.each do |image| %>
 <% path = image.split(":").first; tags = image.split(":").last %>
-<% next if no_built_x86_64_images.any? do |component| tags.include?(component) end %>
 - `<%= tags.split(",").first.split("-")[2].capitalize %>` [Dockerfile](docker-image/<%= path %>/Dockerfile)
+<% next if no_built_x86_64_images.any? do |component| tags.include?(component) end %>
 <% tags.split(",").each do |tag| %>
```

**Comment by kenhys** [comment]
> But I think that x86_64 and arm64 section still needs to display Dockerfile paths for non built images.
What do you think?

This sounds good!

**Comment by cosmo0920** [comment]
@kenhys I pushed the above diff changes. If you're OK, let's merge this OR. :muscle:

**Comment by kenhys** [comment]
Thanks @cosmo0920 !

**Pull request merged**